src/assets/styles/_globals.scss

```diff
@@ -0,0 +1,9 @@
+//
+// Warning: Make sure this manifest never outputs CSS
+// and only import variables, mixins, functions etc.
+// as this file is included in _every_ component
```

Comment:
Thanks, I may get to this, looks a bit cleaner and consistent. But for now this setup is working :)